app-localize-behavior.html

```diff
@@ -247,9 +287,34 @@
     };
   },
 
-  __onRequestResponse: function(event) {
-    this.resources = event.response;
-    this.fire('app-localize-resources-loaded', event, { bubbles: this.bubbleEvent});
+  __onRequestResponse: function(event, language, merge) {
+    var propertyUpdates = {};
+    var newResources = event.response;
+    if (merge) {
+      if (language) {
+        propertyUpdates.resources = Object.assign({}, this.resources || {});
+        propertyUpdates['resources.' + language] = Object.assign(
+          propertyUpdates.resources[language] || {}, newResources);
+      } else {
+        propertyUpdates.resources = Object.assign(this.resources, newResources);
+      }
+    } else {
+      if (language) {
+        propertyUpdates.resources = {};
+        propertyUpdates.resources[language] = newResources;
+        propertyUpdates['resources.' + language] = newResources;
+      } else {
+        propertyUpdates.resources = newResources;
+      }
+    }
+    if (this.setProperties) {
+      this.setProperties(propertyUpdates);
```

Comment:
wait, what is `setProperties`? Is this JavaScript magic I don't know about?

Reply:
new in polymer 2! https://www.polymer-project.org/2.0/docs/devguide/model-data#set-property

Reply:
oh TIL it's Polymer magic: https://www.polymer-project.org/2.0/docs/devguide/model-data#set-property

Reply:
jinx!